packages/api/src/derives/nft/openCollectionListings.ts

```diff
@@ -89,3 +90,72 @@ export function openCollectionListings(instanceId: string, api: ApiInterfaceRx)
     );
   };
 }
+
+/**
+ * Gets all tokens in a collection that have an open listing
+ *
+ * @param collectionId  The collection Id value
+ *
+ * @returns List of tokens that have open listings in given collection in the format {listingId, tokens: vec<DeriveTokenInfo>}
+ */
+export function openCollectionListingsV2(instanceId: string, api: ApiInterfaceRx) {
```

Comment:
so this new version won't bring back the `attributes` anymore? in that case for older runtime version, they have to do another query to return the `attributes`?

Reply:
since we're deprecating onchain attributes, can skip this query.. for older chain, they would need to query at that blockHash.. when the runtime is upgraded the call to get attributes will fail, api.query.nft.seriesAttribute as there s no more seriesAttribute in the storage

Reply:
Yup, that's fine, you might want to note that down for the upgrade guide